scripts/update-dependencies/UpdateFilesTargets.cs

```diff
@@ -190,7 +190,7 @@ private static IEnumerable<JProperty> FindAllDependencyProperties(JObject projec
         [Target]
         public static BuildTargetResult ReplaceCrossGen(BuildTargetContext c)
         {
-            ReplaceFileContents(@"build_projects\dotnet-cli-build\CompileTargets.cs", compileTargetsContent =>
+            ReplaceFileContents(@"build_projects\shared-build-targets-utils\BuildPackageVersions.cs", compileTargetsContent =>
```

Comment:
fixed.

---
In reply to: [64609110](https://github.com/dotnet/cli/pull/3230#discussion_r64609110) [](ancestors = 64609110)